website/sidebars.js

```diff
@@ -76,6 +76,7 @@ module.exports = {
             "docs/guides/migration-guide/upgrading-to-0-16-0",
             "docs/guides/migration-guide/upgrading-to-0-17-0",
             "docs/guides/migration-guide/upgrading-to-0-18-0",
+            "docs/guides/migration-guide/upgrading-to-0-19-0",
```

Comment:
how do you feel about reverse-ordering these?

Reply:
i love it!